conanfile.txt

```diff
@@ -48,5 +48,4 @@ boost:without_context=True
 [imports]
 bin, *.dll -> ./lib # Copies all dll files from packages bin folder to my "lib" folder
 lib, *.dylib* -> ./lib # Copies all dylib files from packages lib folder to my "lib" folder
-lib, *.so* -> ./lib # Copies all dylib files from packages lib folder to my "lib" folder
-lib, *.a* -> ./lib
+lib, *.so* -> ./lib # Copies all dylib files from packages lib folder to my "lib" folder
```

Comment:
Why are we no longer copying the archive files? I believe we do static linking in some places. Although, I guess if this works it doesn't matter.

Reply:
The static libraries are found and taken from their original location. So we don't actually need to copy then to the lib folder (which is meant for the dynamic libraries only)